client_tools.go

```diff
@@ -82,17 +108,26 @@ const (
 
 // Operations is a collection of update operations.
 type Operations struct {
-	ops []Op
+	ops []operation
+}
+
+func (ops *Operations) EncodeMsgpack(enc *msgpack.Encoder) error {
+	return enc.Encode(ops.ops)
 }
```

Comment:
Let's add a unit-test for the new public method.

Reply:
New unit-test added to the `client_tools_test.go`.